conclusions/rq2.md

```diff
@@ -0,0 +1,13 @@
+## What can be learned from existing efforts to involve a community with software development?
+In short, there are a few key factors that can be identified in succesful software development communities:
+- a small core team that actively communicates about the current and desired state of the software;
+- use of GitHub (git, pull-requests, issue-tracker) or a similar system;
+- modular code with standartized documentation, easy set-up and tutorials;
+- use both peer reviews as formal testing;
+- further stimulation of intrinsic and extrinsic motivation.
+
+A small core team is necessary to provide a stable basis on which the rest of the community can rely on. This team should contain members respected by the community and provide clear information about the state and future of the project. In relation to FeedbackFruits, this team can, consist of one person from FeedbackFruits and two or three persons from the community. The members from the community can change per project and at least project issuer should take a place in the team.
```

Comment:
`In relation to FeedbackFruits, this team can, consist of one`

Why the second comma? The first can be left out as well